analyses/WA_cd_2010/03_sim_WA_cd_2010.R

```diff
@@ -0,0 +1,38 @@
+###############################################################################
+# Simulate plans for `WA_cd_2010`
+# © ALARM Project, July 2022
+###############################################################################
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg WA_cd_2010}")
+
+constr <- redist_constr(map) %>%
+    add_constr_grp_hinge(8.0, vap - vap_white, vap, c(0.52, 0.35, 0.25)) %>%
+    add_constr_grp_hinge(-8.0, vap - vap_white, vap, c(0.35, 0.25))
```

Comment:
Can we try increasing the strengths to 10 or 12 here and seeing if that helps with the minority opportunity /VRA district?

Reply:
Tried both and the minority opportunity/VRA district didn't really move. Here is the analysis
![WA 12 0 constraint](https://user-images.githubusercontent.com/57640740/214679370-0a1fd967-6a9f-4f19-ad19-f54f4e4eea2a.png)

Reply:
I also had to up the number of sims to 5k over 2 runs to reach convergence.

Reply:
@CoryMcCartan If upping the number of sims didn't work, should I keep my current constraint (8, -8)?

Reply:
yeah I guess so